src/components/sys-footerbar.vue

```diff
@@ -104,62 +104,79 @@
             Terms &amp; Conditions
           </component>
         </div>
-
-        <div :class="$style.socials">
-          <a
-            aria-label="Facebook"
-            href="https://facebook.com/system76/"
-            rel="noopener"
-            target="_blank"
-            title="Facebook"
-            :class="$style.social"
-          >
-            <font-awesome-icon :icon="faFacebook" />
-          </a>
-
-          <a
-            aria-label="GitHub"
-            href="https://github.com/system76"
-            rel="noopener"
-            target="_blank"
-            title="GitHub"
-            :class="$style.social"
-          >
-            <font-awesome-icon :icon="faGithub" />
-          </a>
-
-          <a
-            aria-label="Instagram"
-            href="https://instagram.com/system76_com"
-            rel="noopener"
-            target="_blank"
-            title="Instagram"
-            :class="$style.social"
-          >
-            <font-awesome-icon :icon="faInstagram" />
-          </a>
-
-          <a
-            aria-label="Telegram"
-            href="https://telegram.me/system76_news"
-            rel="noopener"
-            target="_blank"
-            title="Telegram"
-            :class="$style.social"
-          >
-            <font-awesome-icon :icon="faTelegram" />
-          </a>
-
-          <a
-            aria-label="Twitter"
-            href="https://twitter.com/system76/"
-            rel="noopener"
-            target="_blank"
-            title="Twitter"
-            :class="$style.social"
-          >
-            <font-awesome-icon :icon="faTwitter" />
-          </a>
+        <div :class="$style.connections">
+          <div>
+            <div
+              v-if="showNewsletterButton"
+              :class="$style.subscribe"
+            >
+              <p>Subscribe for updates on products, OS features, and promotions.</p>
+              <p>
+                <button
+                  class="button"
+                  @click.prevent="$emit('action')"
```

Comment:
I'd probably change the event name to something like `subscribe`. `action` is pretty generic and doesn't say what kind of thing actually happened